tests/scr-readingsystem-support_iframe_svg/OPS/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Scripting</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-12-10</dc:date>
+    <dc:description>Checks whether the reading system implements the epubReadingSystem object in container-constraint document.</dc:description>
```

Comment:
This seems to be the same description as scr-readingsystem-support_iframe/OPS/package.opf.

Reply:
This has changed under your feet:-) I think it is better if I stop now (it is late afternoon anyway) and pick this up tomorrow or wherever you did your next batch :-)